.github/workflows/ts_serialize_tests.yml

```diff
@@ -40,12 +40,12 @@ jobs:
         run: deno cache test_deps.ts
 
       - name: Deno test
-        run: deno test --doc
+        run: deno test --doc -c tsconfig.json
```

Comment:
when running deno tests, we need to read the tsconfig to enable experimental decorators.

Otherwise we get an angry typescript 
```
TS1240 [ERROR]: Unable to resolve signature of property decorator when called as an expression.
  Argument of type 'undefined' is not assignable to parameter of type 'Object'.
      @SerializeProperty()
```

on every decorator.

Reply:
see my comment on vscode settings `deno/deno.config`